tests/utils.schemas.tests.js

```diff
@@ -1,57 +1,62 @@
 import { expect } from "chai";
 
-import { buildDependencies, getSchemaWithDependencies, typeofSchema } from "../src/utils/schemas";
+import {
+    buildDependenciesEnum,
+    buildDependenciesOneOf,
+    getSchemaWithDependencies,
+    typeofSchema,
+} from "../src/utils/schemas";
 
 describe("RJSF schema", () => {
     const TREE = {
         path: "/dft",
-        dataSelector: { key: "type", value: "data.type.slug", name: "data.type.name" },
+        dataSelector: { key: "type", value: "data.type.enum[0]", name: "data.type.title" },
         data: {
             type: {
-                slug: "dft",
-                name: "Density Functional Theory",
+                enum: ["dft"],
+                title: "Density Functional Theory",
             },
         },
         children: [
             {
                 path: "/dft/lda",
                 dataSelector: {
                     key: "subtype",
-                    value: "data.subtype.slug",
-                    name: "data.subtype.name",
+                    value: "data.subtype.enum[0]",
+                    name: "data.subtype.title",
                 },
                 data: {
                     subtype: {
-                        slug: "lda",
-                        name: "LDA",
+                        enum: ["lda"],
+                        title: "LDA",
                     },
                 },
                 children: [
                     {
                         path: "/dft/lda/svwn",
                         dataSelector: {
                             key: "functional",
-                            value: "data.functional.slug",
-                            name: "data.functional.name",
+                            value: "data.functional.enum[0]",
+                            name: "data.functional.title",
                         },
                         data: {
                             functional: {
-                                slug: "svwn",
-                                name: "SVWN",
+                                enum: ["svwn"],
+                                title: "SVWN",
                             },
                         },
                     },
                     {
                         path: "/dft/lda/pz",
                         dataSelector: {
                             key: "functional",
-                            value: "data.functional.slug",
-                            name: "data.functional.name",
+                            value: "data.functional.enum[0]",
+                            name: "data.functional.title",
```

Comment:
Per our discussion:

1. Let's add type to know whether we are dealing with a string or object for enum[0] use
2. When object, we can have `value: data.functional`, `name: data.functional.name`